src/data/faq.json

```diff
@@ -2239,19 +2477,29 @@
             ],
             "V": [
                 {
-                    "term": "Validity of an EU digital COVID certificate",
+                    "term": "Validity of an EU Digital COVID Certificate",
                     "anchor": "validity",
                     "description": "Basically, we distinguish between <ul><li>the validity of the digital certificate (the technical validity) and</li><li>the validity of the vaccination protection (the effective validity).</li></ul>The digital certificate has an issue date and a technical expiration date (so does your passport and PA). The validity of the certificate ends 1 year after issuance. <br/><br/> <b>Vaccination Certificate</b> <br/><br/>In the certificate data is transported, in this case information about vaccination. This vaccination took place on a date X and is valid from then (according to the current state of scientific knowledge) e.g. in Germany 1 year. After that the vaccination protection has expired. We call the duration of the vaccination protection 'effective (vaccination) validity'. So the effective validity of the vaccination counts from the last vaccination date (date X).  <br/><br/> A certificate is valid until either the technical validity or the effective validity expires."
                 },
                 {
-                    "term": "Vaccination certificate",
+                    "term": "Vaccination Certificate",
                     "anchor": "vaccination_certificate",
-                    "description": "The vaccination certificate is a <a href='#glossary_digital_certificate'>digital certificate</a> issued for a person who has received a vaccination with a vaccine approved in the EU. Each vaccination certificate documents exactly one vaccination event (one dose). <br/><br/>The vaccination certificate represents a digital form of a <a href='#glossary_vaccination_proof'>proof of vaccination</a>."
+                    "description": "The vaccination certificate is a <a href='#glossary_digital_certificate'>digital certificate</a> issued for a person who has received a vaccination with a vaccine approved in the EU. Each vaccination certificate documents exactly one vaccination event (one dose). <br/><br/>The vaccination certificate represents a digital form of a <a href='#glossary_proof_of_vaccination'>proof of vaccination</a>."
+                },
+                {
+                    "term": "Vaccination Certificate Token",
+                    "anchor": "vaccination_certificate_token",
+                    "description": "A QR code generated by vaccination centers or doctors' offices that can be scanned in the Corona alert app, for example, to document a vaccination."

```

Comment:
"Corona alert app" => "Corona-Warn-App"

Reply:
Will be changed in the next commit, thanks!